### PR TITLE
adding beta feature error message

### PR DESCRIPTION
**Pull request by hermankailey**
adding error message for featureUtility feature install on beta runtimes.

### CODE REVIEW

**Comment by hermankailey** [comment]
fixes #12224

File: dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops [hunk 1]
```diff
@@ -870,6 +870,10 @@ ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY=CWWKF1410E: The {0} feature cannot
 ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY.explanation=The feature is not applicable to the edition of the installed product.
 ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY.useraction=Install features that are supported by the product edition.
 
+ERROR_BETA_EDITION_NOT_SUPPORTED=CWWKF1411E: Installing or updating features is not supported for Beta releases.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops:873]: "Beta" doesn't need to be capitalized (unless it is throughout the file or something like that).

Comment (hermankailey) [reply dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops]: change made, thanks!

**Comment by hermankailey** [comment]
#build

**Comment by hermankailey** [comment]
#build

**Comment by andrea-pichler** [comment]
Completed the requested L2 message review

**Comment by hermankailey** [comment]
#run-libby-bot

**Pull request merged**